strings.go

```diff
@@ -0,0 +1,24 @@
+package gana
+
+import "strings"
+
+// DropString allocates a new string, with the first `num` bytes of a string dropped.
+func DropString(num uint, what string) string {
+	l := uint(len(what))
+	if l < num {
+		return ""
+	}
+
+	builder := strings.Builder{}
+	builder.WriteString(what[num:])
+
+	return builder.String()
+}
+
+// SkipString skips the first `num` bytes of a string by slicing (underlying array unaffected).
+func SkipString(num uint, what string) string {
+	if uint(len(what)) < num {
```

Comment:
Yeah, like comment above, let's keep the allocation of a static variable consistent -- if it's used multiple times in the definition, then make one, otherwise, if it's only used for a range check, do it inline.